README.md

```diff
@@ -31,34 +31,50 @@ critical things like tuning performance and getting the most out of
 Spanner. Expect that you'll need to tweak and enhance what HarbourBridge
 produces.
 
-To use the tool on a PostgreSQL database called mydb, run
+Below are some quick starter examples on how to run HarbourBridge. You can look
+at specific sections for PostgreSQL, MySQL etc for detailed examples.
+
+For example, to generate Spanner schema from a PostgreSQL dump file, run
 
 ```sh
-# By default, the driver is "pg_dump".
-pg_dump mydb | harbourbridge
-# Or,
-pg_dump mydb | harbourbridge -driver=pg_dump
+# Generate pg_dump file.
+pg_dump mydb > mydb.pg_dump
```

Comment:
Am wondering why not keep the original command structure and just pipe the output of pg_dump to HarbourBridge? i.e. something like:

pg_dump mydb | harbourbridge schema -source=postgresql

Or do you think that the dump file is the more common usage and we should focus on that?

Reply:
From a performance point of view, it's not ideal to pipe rather large dump files to HarbourBridge since stdin provided in this manner is not seekable and we end up creating a copy. I have thus tried to move away from providing examples where we suggest to just pipe the output for pg_dump to HB. I have though deleted these examples altogether as explained in an earlier reply.